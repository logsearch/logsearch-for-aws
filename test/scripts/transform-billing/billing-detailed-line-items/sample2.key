123456789012-aws-billing-detailed-line-items-with-resources-and-tags-2015-07.csv.zip
