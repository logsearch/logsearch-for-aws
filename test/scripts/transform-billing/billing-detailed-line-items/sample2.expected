{"InvoiceID":12345678,"PayerAccountId":123456789012,"LinkedAccountId":123456789012,"RecordType":"LineItem","RecordId":"30000000000000000000000001","ProductName":"Amazon Simple Queue Service","RateId":3304352,"SubscriptionId":16468634,"PricingPlanId":495777,"UsageType":"USW2-Requests-Tier1","Operation":"DeleteMessage","AvailabilityZone":null,"ReservedInstance":"N","ItemDescription":"First 1,000,000 Amazon SQS Requests per month are free","UsageStartDate":"2015-07-01 00:00:00","UsageEndDate":"2015-07-01 01:00:00","UsageQuantity":95.0,"Rate":0.0,"Cost":0.0,"ResourceId":null,"user":{"deployment":null,"director":null}}
{"InvoiceID":12345678,"PayerAccountId":123456789012,"LinkedAccountId":123456789012,"RecordType":"LineItem","RecordId":"30000000000000000000000002","ProductName":"Amazon CloudFront","RateId":3506629,"SubscriptionId":16468636,"PricingPlanId":504635,"UsageType":"AP-DataTransfer-Out-Bytes","Operation":"GET","AvailabilityZone":null,"ReservedInstance":"N","ItemDescription":"$0.140 per GB - first 10 TB / month data transfer out","UsageStartDate":"2015-07-01 00:00:00","UsageEndDate":"2015-07-01 01:00:00","UsageQuantity":0.0103827,"Rate":0.14,"Cost":0.00145358,"ResourceId":"E34Y093QOCD746","user":{"deployment":null,"director":null}}
{"InvoiceID":12345678,"PayerAccountId":123456789012,"LinkedAccountId":123456789012,"RecordType":"LineItem","RecordId":"30000000000000000000000003","ProductName":"Amazon Elastic Compute Cloud","RateId":5831118,"SubscriptionId":38594944,"PricingPlanId":646792,"UsageType":"EUC1-BoxUsage:t2.micro","Operation":"RunInstances","AvailabilityZone":"eu-central-1a","ReservedInstance":"N","ItemDescription":"$0.015 per On Demand Linux t2.micro Instance Hour","UsageStartDate":"2015-07-01 00:00:00","UsageEndDate":"2015-07-01 01:00:00","UsageQuantity":1.0,"Rate":0.015,"Cost":0.015,"ResourceId":"i-03013fcd","user":{"deployment":"infra/core","director":"prod-abraxas-aws-euc1"}}
{"InvoiceID":12345678,"PayerAccountId":123456789012,"LinkedAccountId":123456789012,"RecordType":"LineItem","RecordId":"30000000000000000000000004","ProductName":"Amazon Elastic Compute Cloud","RateId":3510857,"SubscriptionId":38594942,"PricingPlanId":505699,"UsageType":"USW2-DataTransfer-Regional-Bytes","Operation":"InterZone-Out","AvailabilityZone":null,"ReservedInstance":"N","ItemDescription":"$0.010 per GB - regional data transfer - in/out/between EC2 AZs or using IPs or ELB","UsageStartDate":"2015-07-01 00:00:00","UsageEndDate":"2015-07-01 01:00:00","UsageQuantity":0.06222631,"Rate":0.01,"Cost":0.00062226,"ResourceId":"i-27d59dd0","user":{"deployment":"elasticsearch-publicweb","director":"prod-abraxas-aws-usw2"}}
